packages/ui/src/components/va-accordion/hooks/useAccordion.ts

```diff
@@ -79,11 +81,15 @@ export const useAccordionItem = (state: WritableComputedRef<boolean>) => {
 
   const item = { state }
 
-  onMounted(() => accortion.onItemMounted(item))
-  onBeforeUnmount(() => accortion.onItemUnmounted(item))
-  watch(state, () => accortion.onItemChanged(item))
+  onMounted(() => accordion.onItemMounted(item))
+  onBeforeUnmount(() => accordion.onItemUnmounted(item))
 
   return {
-    accordionProps: accortion.props,
+    accordionProps: accordion.props,
+
+    toggle: () => {
```

Comment:
Simple function names usually require comments.

```suggestion
    /** Toggle collapse value and notify accordion about it */ 
    toggle: () => {
```